ftp/src/test/scala/akka/stream/alpakka/ftp/CommonFtpStageSpec.scala

```diff
@@ -71,7 +71,7 @@ trait CommonFtpStageSpec extends BaseSpec with Eventually {
   implicit val system = getSystem
   implicit val mat = getMaterializer
   implicit val defaultPatience =
-    PatienceConfig(timeout = Span(10, Seconds), interval = Span(300, Millis))
+    PatienceConfig(timeout = Span(30, Seconds), interval = Span(600, Millis))
```

Comment:
Might be wise, I've seen timeouts as well. Are those actually to be expected? Do we know why those are so slow?